src/coreclr/src/pal/src/misc/sysinfo.cpp

```diff
@@ -565,9 +565,13 @@ PAL_GetLogicalProcessorCacheSizeFromOS()
     cacheSize = std::max(cacheSize, (size_t)sysconf(_SC_LEVEL4_CACHE_SIZE));
 #endif
 
-#if defined(HOST_ARM64)
-    if(cacheSize == 0)
+    if (cacheSize == 0)
     {
+        //
+        // Fallback to retrieve cachesize via /sys/.. if sysconf was not available 
+        // for the platform. Currently musl and arm64 should be only cases to use  
+        // this method to determine cache size.
+        // 
         size_t size;
 
         if(ReadMemoryValueFromFile("/sys/devices/system/cpu/cpu0/cache/index0/size", &size))
```

Comment:
I assume we will never hit this for windows, but I dont see any special casing just for linux here.

Reply:
This is in PAL, it is never compiled for Windows.